src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java

```diff
@@ -1089,6 +1090,11 @@ public void initializeGridIndex(final Set<String> countries)
         }
     }
 
+    public void setForceSlicingPredicate(final Predicate<Taggable> forceSlicingPredicate)
+    {
+        this.forceSlicingPredicate = forceSlicingPredicate;
```

Comment:
Name doesn't really indicate what this predicate is testing. A name like `shouldSlice` or `shouldNotSlice` could be more clear.

Reply:
Sounds good!